package.json

```diff
@@ -61,6 +65,7 @@
 		"eslint": "^8.43.0",
 		"eslint-plugin-prettier": "^4.2.1",
 		"eslint-plugin-vue": "^9.15.0",
+		"file-saver": "^2.0.5",
```

Comment:
You can simply use `URL.createObjectURL()`, create a link element and trigger a click on it without any extra dependency needed.

Reply:
The rest seems fine, just clear out this unnecessary dep.